### PR TITLE
Create PRS13456Nurit.xml

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

File: new/PRS13456Nurit.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2021-07-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName xml:lang="gez" xml:id="n1">ኑሪት፡</persName>
```
Comment (DenisNosnitsin1970) [inline new/PRS13456Nurit.xml:48]: She was/is a Muslim, most probably.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/PRS13456Nurit.xml (hunk 1 above)
Comment (thea-m) [reply new/PRS13456Nurit.xml:48]: I agree, but the scroll has Christian content, so I didn't decide for any faith

**Comment by DenisNosnitsin1970** [comment]
I think that it is not so rigid with magic texts, and it is said that elements of the magic are supra-ethnic and supra-confessional, but somehow tangible proofs were missing. In other words, I would not be amaized if an Ethiopian Muslim uses a Christian amulet, and vice versa. I think It would be useful, to indicate, at least hypothetically, that she was a Muslim.

**Comment by thea-m** [comment]
Yes, there are apparently several cases, also described by Veronika Six, see also https://betamasaheft.eu/manuscripts/EMIPms00159/main (it would be interesting if in this latter case, both names occur in the same stratum and refer to the same person or whether one was added later)

**Pull request merged**